apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/model/oms/OmstillingsstoenadInnvilgelse.kt

```diff
@@ -81,7 +92,7 @@ data class OmstillingsstoenadInnvilgelse(
                     avdoed.doedsdato
                         .plusMonths(4)
                         .isAfter(avkortingsinfo.virkningsdato),
-                lavEllerIngenInntekt = brevutfall.lavEllerIngenInntekt == LavEllerIngenInntekt.JA,
+                omsRettUtenTidsbegrensning = omsRettUtenTidsbegrensning.hovedvilkaar.resultat == Utfall.OPPFYLT,
```

Comment:
Har du en tilsvarende endring på pensjonsbrev for å kunne mappe opp dette parameteret som nå har fått nytt navn?

Reply:
Ja, har rullet ut en endring i pensjonsbrev som tillater begge verdier midlertidig ref. https://github.com/navikt/pensjonsbrev/pull/761